homeassistant/components/switch/broadlink.py

```diff
@@ -209,8 +263,13 @@ def should_poll(self):
     @property
     def is_on(self):
         """Return true if device is on."""
+        self.update()
         return self._state
 
+    def update(self):
+        """Synchronize state with switch."""
+        pass
```

Comment:
Why do you need this??

Reply:
Define a empty function for `is_on(self)` to call even override by sub module
sorry for my `JAVA` like programming

Reply:
I am trying to find `update(self)` function in [switch/SwitchDevice](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/switch/__init__.py#L137)
and find nothing.
if it's passable to make it like this

```
    @abstractmethod
    """Synchronize state with switch."""
    def update(self):
```

Reply:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py#L171

Reply:
I see. I will remove this soon.